aws/transport/http/request_id.go

```diff
@@ -0,0 +1,40 @@
+package http
```

Comment:
Not sure if this will or won't be an HTTP specific concept, so far its been generic across AWS request.